qiskit/visualization/latex.py

```diff
@@ -275,8 +275,14 @@ def _get_image_depth(self):
         # wires in the beginning and end
         columns = 2
 
-        # all gates take up 1 column except from those with labels (cu1) which take 2
-        columns += sum([2 if nd.name == 'cu1' else 1 for layer in self.ops for nd in layer])
+        # all gates take up 1 column except from those with labels (ie cu1)
+        # which take 2 columns
+        for layer in self.ops:
+            column_width = 1
+            for nd in layer:
+                if nd.name == 'cu1':
```

Comment:
i think `rzz` is similar.
See: https://github.com/Qiskit/qiskit-terra/pull/3155

Reply:
Yep, it is. I'll add it to the check here. As an aside we should push that change upstream into qcircuit